package.json

```diff
@@ -21,14 +28,19 @@
   },
   "homepage": "https://github.com/c3c1l1a/Webpack-exercise#readme",
   "devDependencies": {
+    "@babel/core": "^7.18.2",
+    "@babel/preset-env": "^7.18.2",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^28.1.1",
     "css-loader": "^6.7.1",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
     "hint": "^6.2.0",
     "html-loader": "^3.1.0",
     "html-webpack-plugin": "^5.5.0",
+    "jest-environment-jsdom": "^28.1.1",
+    "jest-localstorage-mock": "^2.4.21",
```

Comment:
**[REQUIRED CHANGES]**

- Hey Team, you've done a great job with the testing however the jest testing dependencies are missing in your `package.json` and `package-lock.json`. I had to install it before I could run the test. Please install jest using 
the `npm install jest` command to update the dependencies. see the images below:

*package.json*
![jest](https://user-images.githubusercontent.com/59397159/173090529-3a35211f-638e-44a0-a621-c576dd654a09.png)

*package-lock.json*
![jestpack-lock json](https://user-images.githubusercontent.com/59397159/173090526-e4f9e454-43eb-4b78-befc-7f71c5eb8068.png)

Reply:
Well resolved 👍🏽